lib/browser/objects-registry.js

```diff
@@ -28,11 +28,9 @@ class ObjectsRegistry {
       owner = this.owners[webContentsId] = new Set()
       this.registerDeleteListener(webContents, webContentsId)
     }
-    if (!owner.has(id)) {
```

Comment:
Thanks for the explanation, I will fix the PR to ensure render-view-deleted is emitted/handled properly in sandbox